src/Sorting.java

```diff
@@ -0,0 +1,61 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class Sorting {
+
+    private List<Player> players = new ArrayList<>();
+
+    public List<Player> getPlayers() {
+        return players;
+    }
+
+    public void setPlayers(List<Player> players) {
+        this.players = players;
+    }
+
+    public void add(Player player) {
+        players.add(player);
+    }
+
+    public void sortByNameUp() {
+        Collections.sort(players);
+        players.forEach(System.out::println);
```

Comment:
Don't print in sorting methods. Name says sortBy... it doesn't say sortAndPrint. First sort and then print after invoking sorting method.